components/resources/brave_vpn_strings.grdp

```diff
@@ -279,4 +279,36 @@ However, we noticed this helper service is not running - and we don't want to le
   <message name="IDS_BRAVE_VPN_MAIN_PANEL_SESSION_EXPIRED_PART_CONTENT" desc="Content text for session expired panel">
     Please log in to your <ph name="BRAVE_ACCOUNT_BEGIN">$1</ph>Brave Account<ph name="BRAVE_ACCOUNT_END">$2</ph> to refresh your VPN session.
   </message>
+  <if expr="is_win">
+    <message name="IDS_BRAVE_VPN_WIREGUARD_TRAY_ICON_TOOLTIP_DISCONNECTED" desc="Message to be displayed as tooltip for the system tray icon in disconnected state.">
+      Brave VPN: Disconnected
+    </message>
+    <message name="IDS_BRAVE_VPN_WIREGUARD_TRAY_ICON_TOOLTIP_CONNECTED" desc="Message to be displayed as tooltip for the system tray icon in connected state.">
+      Brave VPN: Connected
+    </message>
+    <message name="IDS_BRAVE_VPN_WIREGUARD_TRAY_ICON_TOOLTIP_ERROR" desc="Message to be displayed as tooltip for the system tray icon in error state.">
+      Brave VPN: Error
+    </message>
+    <message name="IDS_BRAVE_VPN_WIREGUARD_TRAY_STATUS_ITEM_ACTIVE" desc="Message to be displayed as the first item in the menu for the system tray icon in connected state.">
+      Status: Active
+    </message>
+    <message name="IDS_BRAVE_VPN_WIREGUARD_TRAY_STATUS_ITEM_INACTIVE" desc="Message to be displayed as the first item in the menu for the system tray icon in disconnected state.">
+      Status: Inactive
+    </message>
```

Comment:
Should we use more consistent language here? The menu commands are `Connect` and `Disconnect`, the tooltip strings are `Brave VPN: Connected` and `Brave VPN: Disconnected`. Would it be better for the status to also say `Status: Connected` and `Status: Disconnected`. Otherwise what's the difference between Connected and Active and Disconnected and Inactive? cc: @rmcfadden3

Reply:
sure, updated to `Status: Connected` and `Status: Disconnected`, thanks